client/app/components/ApplicationArea/ApplicationLayout/DesktopNavbar.jsx

```diff
@@ -165,6 +166,7 @@ export default function DesktopNavbar() {
         <Menu.SubMenu
```

Comment:
This item also does not have keyboard accessibility, but it has a `tabIndex`. We need to track this issue, too.